application/ai/embedding_config_service.py

```
@@ -78,9 +78,17 @@ def _get_db(self):
         """获取数据库连接（延迟导入避免循环依赖）。"""
         if self._db is not None:
             return self._db
-        from infrastructure.persistence.database.connection import get_database
-
-        return get_database()
+        try:
+            from interfaces.api.dependencies import get_db as _get_global_db
+            return _get_global_db()
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)